### PR TITLE
Update momentum.md

### PR DESCRIPTION
**Pull request by BalajiAI**
In the summary section of this file (last point), v isn't momentum, v should be velocity. The parameter (beta) is called momentum.
See here, for more details: https://pytorch.org/docs/stable/generated/torch.optim.SGD.html

*Description of changes:*
momentum ---> velocity

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by BalajiAI** [comment]
@astonzhang Kindly review!

**Comment by BalajiAI** [comment]
@astonzhang Kindly review!

**Comment by astonzhang** [comment]
@BalajiAI Thanks! Could you send another PR to replace your github id with your name in our acknowledgement:
https://github.com/d2l-ai/d2l-en/blob/master/chapter_preface/index.md

**Comment by astonzhang** [comment]
Besides, I made further changes for consistency: https://github.com/d2l-ai/d2l-en/commit/4d3d07f411a60ccbba3f6fd502baa0b2f09f5828

Just let me know if it looks good to you.

**Comment by BalajiAI** [comment]
@astonzhang Tensorflow docs also seems nice.

Everything is good in the mentioned commit, but you forget to add velocity after removing momentum at line 278.
So I made another PR to fix it (https://github.com/d2l-ai/d2l-en/pull/2230).

**Comment by BalajiAI** [comment]
@astonzhang Also another thing. The title is also called momentum. It should be originally called velocity. Since momentum is just another hyperparameter and velocity is the one which stores the leaky averages of the past gradients.
I mentioned this because in the book, you spoke about CNN's are performing cross-correlation and not convolution (strictly speaking).
